src/components/progress-report/variance-explanation/variance-explanation.repository.ts

```diff
@@ -39,6 +39,7 @@ export class ProgressReportVarianceExplanationRepository extends DtoRepository(
   protected hydrate() {
     const defaults: UnsecuredDto<VarianceExplanation> & ExpressionInput = {
       report: 'report.id' as ID,
+      deprecated: [],
```

Comment:
Again not needed. This deprecated list is only for the options, not for actual saved values for reports.